molecule/ccloud/prepare.yml

```diff
@@ -41,7 +41,7 @@
 
     - name: Copy CA Cert to Host
       copy:
-        src: "../../../../generated_ssl_files/{{ssl_self_signed_ca_cert_filepath|basename}}"
+        src: "../../generated_ssl_files/{{ssl_self_signed_ca_cert_filepath|basename}}"
```

Comment:
What has happened here? Has the file been moved?

Reply:
this is a bug left behind from the molecule dir move PR, it went down two directories.
i consider this a one off